core/design-system/src/jvmMain/kotlin/team/aliens/dms/kmp/core/designsystem/webview/DmsWebView.jvm.kt

```
@@ -0,0 +1,8 @@
+package team.aliens.dms.kmp.core.designsystem.webview
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+@Composable
+actual fun DmsWebView(modifier: Modifier, url: String) {
+}
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)